docs/guides/high-availability/examples/generated/ispn-single.yaml

```diff
@@ -25,6 +25,17 @@ data:
           namesAsTags: true
           gauges: true
           histograms: false
+      server:
```

Comment:
@ahus1 We should not expose this in the production environment. This is needed only for our testing. I think we should include some production switch to our Helm chart. WDYT?

Reply:
While it is generated in the big file, we never show this part in the docs. Feel free to go ahead and update the helm chart so we can update it in the generation. It might still be enabled by default for our purposes.

Reply:
I see, that makes sense. I will update the helm chart and we can change this later. This is not blocker for merging then